INSTALL.md

```diff
@@ -1,90 +1,81 @@
-# Package Locations
+# Installation
 
-## PackageCloud
+Prebuilt `deb` and `rpm` packages are available via [packagecloud.io](https://packagecloud.io/). These files are also available to download directly, via the [releases page](https://github.com/Instrumental/instrumentald/releases).
```

Comment:
The package cloud link should probably link to `https://packagecloud.io/app/expectedbehavior/instrumental/search?q=instrumentald&filter=`, or at least to the EB link.